Backend/newman-tests/tests/004-registry-apis/post-test.sh

```
@@ -0,0 +1,11 @@
+set -e
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)